### PR TITLE
TP-610 Add measures search filter

### PR DESCRIPTION
**Pull request by DIT-CJWaszczuk**
## Summary 
Add measures list view
## Notes 
- Had to pretty much build the form structure in the form file.
- Couldn't apply a class to the wrapper of some of the form fields, so had to do a janky `quarters` class approach to get the same style. 
## Screenshots 
image

### CODE REVIEW

**Review by stuaxo — APPROVED** [review]

**Comment by DIT-CJWaszczuk** [comment]
Just missing the conditions table field, but it should be reviewable now.

**Review by stuaxo — APPROVED** [review]
LGTM

**Review by stuaxo — APPROVED** [review]

File: footnotes/filters.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
 from footnotes.validators import FOOTNOTE_TYPE_ID_PATTERN
 
 COMBINED_FOOTNOTE_AND_TYPE_ID = re.compile(
-    r"^(?P<footnote_type_id>" + FOOTNOTE_TYPE_ID_PATTERN + ")"
-    r"(?P<footnote_id>" + FOOTNOTE_ID_PATTERN + ")$",
+    r"(?P<footnote_type_id>" + FOOTNOTE_TYPE_ID_PATTERN + ")"
+    r"(?P<footnote_id>" + FOOTNOTE_ID_PATTERN + ")",
```
Comment (andyhd) [inline footnotes/filters.py:20]: It is generally better to use f-strings than the `+` operator to concatenate strings. It won't make much difference here, though.

**Review by andyhd — APPROVED** [review]

File: footnotes/filters.py (hunk 1 above)
Comment (DIT-CJWaszczuk) [reply footnotes/filters.py:20]: think that may have been to comply with `re.compile`, but we can change it up in a different PR.

**Pull request merged**